protecode/model.py

```diff
@@ -168,10 +175,13 @@ def has_binary(self) -> bool:
         return self.raw.get('has_binary')
 
 
-def highest_major_cve_severity(vulnerabilites: Iterable[Vulnerability]) -> int:
+def highest_major_cve_severity(
+    vulnerabilites: Iterable[Vulnerability],
+    protecode_cvss_version=CVSSVersion.V2,
```

Comment:
do not define default CVSS version here. also, do not include `protecode` in the name.